content/projects/gsoc/2019/schedule.adoc

```diff
@@ -0,0 +1,134 @@
+---
+layout: project
+title: "Jenkins year-round GSoC schedule"
+tags:
+- gsoc
+- gsoc2019
+---
+
+= Jenkins year-round GSoC schedule
+
+== AUG 2018 - FEB 2019
+**Call for GSoC project proposals/mentors**
+
+* period of year where we pro-actively seek for project proposals
+* pre-book SMEs, ask them to make a few hours per coding phase in their schedule (even more so if their SIG participates in GSoC)
+* form mentor groups
+* preplanning meetings
+
+**Mentor and project champion recruitment**
+
+* period of the year when we actively search for mentors from various sources like SIGs and mailing list
+
+== AUG - DEC
+**Swag shipping schedule**
```

Comment:
probably could be removed as well

Reply:
@oleg-nenashev Prospective mentors and students may want to know when they should expect swag, as advertised: "P.S: I am going to create a special GSoC 2018 edition of swags for mentors and students. Do not miss them ;)"
(https://jenkins.io/blog/2018/02/19/gsoc2018-announcement/)